lstchain/scripts/lstchain_cherenkov_transparency.py

```diff
@@ -0,0 +1,680 @@
+#!/usr/bin/env python
+
+import glob
+import tables
+import gc
+import numpy as np
+import pandas as pd
+import argparse
+from datetime import datetime
+from scipy.signal import argrelextrema
+from scipy.optimize import curve_fit
+from scipy.stats import chi2
+
+parser = argparse.ArgumentParser(description="compute Cherenkov transparency")
+
+#
+# This script reads in the DL1 datacheck files for all runs which correspond
+# to a given night summary DL1 datacheck file, i.e. DL1_datacheck_YYYYMMDD.h5,
+# calculates some subrun-wise parameters related to the intensity spectrum of
+# cosmic rays (dR/dI, differential rate per unit of intensity), and writes
+# them in a new "cosmics_intensity_spectrum" table in the night summary file.
+#
+# These parameters are very useful to assess the quality of data, and also
+# may eventually be used to apply corrections to the absolute light
+# calibration for nights with smaller-than-optimal atmospheric transmissivity
+#
+# Input files will be the run-wise datacheck files in the path provided
+# through the input_dir command-line argument
+#
+# The night's datacheck summary file is the output_file, because it will be
+# updated by this script
+#
+parser.add_argument('-u', '--update_datacheck_file', dest='output_file',
+                    type=str, default='',
+                    help='DL1_datacheck_YYYYMMDD.h5 file')
+
+parser.add_argument('-d', '--input_dir', dest='input_dir',
+                    type=str, default='v0.10/tailcut84/datacheck',
```

Comment:
This input directory argument could be named `prod_id` or `prod_version` instead. As an input directory, I would rather give the whole path.